cli/dts/lib.deno.ns.d.ts

```diff
@@ -169,6 +169,22 @@ declare namespace Deno {
    */
   export const noColor: boolean;
 
+  /**
+   * Returns the release version of the Operating System.
+   *
+   * ```ts
+   * console.log(Deno.osRelease());
+   * ```
+   *
+   * Requires `allow-env` permission.
+   * Under consideration to possibly move to Deno.build or Deno.versions and if
+   * it should depend sys-info, which may not be desirable.
+   *
+   * @tags allow-env
+   * @category Runtime Environment
+   */
+  export function osRelease(): string;
```

Comment:
This returns `"6.2.9200"` on Windows 10 19043.2006. I probably don't understand windows versioning.

Reply:
I think deno uses the wrong windows api. It uses GetVersionEx and not RtlGetVersion. Thats why it returns the wrong version.

Reply:
Thanks for the heads up, we're gonna stabilize anyway and fix it in a follow up as this API being unstable is a blocker for several Node modules.